DB_Design/Liquibase/dbchangelog.xml

```diff
@@ -1,134 +1,116 @@
- <?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
   <databaseChangeLog
     xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
   
-  <changeSet id=".01" author="MTavares">
+    <changeSet id=".01" author="MTavares">
 
           <createTable tableName="Shooter">
             <column name="shooter_id" type="int">
-                <constraints primaryKey="true" nullable="false"/>
+                <constraints primaryKey="true" nullable="false" />
             </column>
             <column name="first_name" type="varchar(50)">
-                <constraints nullable="false"/>
+                <constraints nullable="false" />
             </column>
             <column name="last_name" type="varchar(50)">
-                <constraints nullable="false"/>
+                <constraints nullable="false" />
             </column>
-            <column name="active" type="boolean"                     
-              defaultValueBoolean="true"/>
-              </column>
-              <column name="iscoach" type="boolean"                     
-              defaultValueBoolean="false"/>
-              </column>
-              <column name="discription" type="varchar(max)">
-                <constraints nullable="false"/>
+            <column name="active" type="boolean" defaultValueBoolean="true" />
+            <column name="iscoach" type="boolean" defaultValueBoolean="false" />
```

Comment:
inconsistent. first_name and last_name, but here we have iscoach. expect is_coach